dev-docs/source/Testing/ReflectometryGUI/ReflectometryGUITests.rst

```diff
@@ -175,7 +175,10 @@ Export as ASCII
   - Amongst other things this text should contain the logs you selected, e.g. ``nperiods : 1`` and ``run_end : 2011-12-15T14:19:13``.
 
 - Try changing the separator to spaces or tabs and check that the 3 or 4 columns of numbers are separated using that separator.
-- Change the dropdown to ``3 column (*.dat)``. The checkboxes, separators and parameter settings are not applicable so they should be greyed out. Click save and you should get 3 columns of numbers separated by tabs (including a leading tab). At the top there is an integer indicating the number of lines in the data.
+- Change the dropdown to ``ORSO Ascii (*.ort)``. The ``Header`` checkbox, separators and parameter settings are not applicable so they should be greyed out.
+- Untick ``Q resolution`` and click Save. You should get a header at the top starting with ``ORSO reflectivity data file``. There should be 3 columns of numbers with headings ``Qz``, ``R`` and ``sR``.
+- Tick ``Q resolution`` and re-save. The file should now contain 4 columns of numbers with headings ``Qz``, ``R``, ``sR`` and ``sQz``.
+- Change the dropdown to ``3 column (*.dat)``. The settings should remain greyed out and the ``Q resolution`` checkbox should also be greyed out as it is not applicable. Click save and you should get 3 columns of numbers separated by tabs (including a leading tab). At the top there is an integer indicating the number of lines in the data.
```

Comment:
dev-docs-html target is built successfully and I have followed the instructions under `Export as ASCII` section which generated all the output as described in my Windows10 machine. 

- While following the instructions as a new user I was a bit confused at the test instruction at line `169` in which after clicking save, how to open that file. Is it meant to open the file via some text editor(which I did) or via the Mantid. 
- On line 179, isn't it worth mentioning the new file created after clicking Save with the `.ort` extension where the expected header and columns would reside? Maybe it's trivial for the existing users to expect a new file with the selected extension.

Reply:
The details on the positioning of the expected lines are indeed clear enough. 

But after clicking `Save` I wasn't sure which file to check with the expected header since there can be a bunch of other files already existing in the save directory configured. Therefore it may be worth mentioning where to check the content (in the new `.ort` file or something like that).

Reply:
Ah, I see. I think on that I'm inclined to leave the instructions as they are for now if that's OK and see how we get on. The instructions for this new file format are added in the same style as the existing ones, which haven't caused any confusion in the past. Between lines 167 and 169 we're effectively giving a tutorial on how to save out and identify files from the interface. The instructions also mention that you're saving the `ORSO Ascii (*.ort)` format, so that indicates the file extension you would be looking for. If we find it causes any confusion then we can update the instructions for all the save formats for the next release.

Reply:
Thanks Rachel for the clarifications!